features/docs/gegeven-stap-definities-expressief.feature

```diff
@@ -0,0 +1,70 @@
+      | 1    | inschrijving   | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING * | 0                       |
+      |      | persoon        | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,voor_naam) VALUES($1,$2,$3,$4,$5,$6)                         | 9999,0,0,P,000000036,P1 |
+      |      | verblijfplaats | INSERT INTO public.lo3_pl_verblijfplaats(pl_id,volg_nr,inschrijving_gemeente_code) VALUES($1,$2,$3)                                                   | 9999,0,0518             |
+
+  Scenario: de persoon '[aanduiding]' met burgerservicenummer '[bsn]' is in Nederland geboren
+    Gegeven de persoon 'P1' met burgerservicenummer '000000036'
+    * is in Nederland geboren
+      Dan zijn de gegenereerde SQL statements
+      | stap | categorie    | text                                                                                                                                                           | values                               |
+      | 1    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *          | 0                                    |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,voor_naam,geboorte_land_code,akte_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8) | 9999,0,0,P,000000036,P1,6030,1AA0100 |
+
+  # ouder 1 is meerderjarig met geboorte_datum [gisteren - 30 jaar]
+  Scenario: de persoon '[aanduiding]' met burgerservicenummer '[bsn]' heeft ouder 1 met burgerservicenummer '[bsn]'
+    Gegeven de persoon 'P1' met burgerservicenummer '000000036'
+    * heeft een ouder 1 'P2' met burgerservicenummer '000000012'
+      Dan zijn de gegenereerde SQL statements
+      | stap | categorie    | text                                                                                                                                                  | values                                  |
+      | 1    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING * | 0                                       |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,voor_naam) VALUES($1,$2,$3,$4,$5,$6)                         | 9999,0,0,P,000000036,P1                 |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geboorte_datum,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6,$7)  | 9999,0,0,P,000000012,19940923,Voorbeeld |
```

Comment:
dit is de tweede persoon die je toevoegt, dus moet een andere pl_id krijgen: 10000

Reply:
voor 1e persoon wordt nu pl_id 9999 gehanteerd, 2e persoon krijgt pl_id 10000 en evt 3e persoon pl_id 10001